python_modules/dagster/dagster/core/storage/runs/sql_run_storage.py

```diff
@@ -143,11 +145,6 @@ def handle_run_event(self, run_id: str, event: DagsterEvent):
         if event.event_type not in EVENT_TYPE_TO_PIPELINE_RUN_STATUS:
             return
 
-        run = self.get_run_by_id(run_id)
```

Comment:
I wonder how many times we fetch the run from the db per event write haha. Now that -1